app/pages/propertyrentals/ClaimPropertyIncomeAllowancePage.scala

```diff
@@ -16,14 +16,29 @@
 
 package pages.propertyrentals
 
-import models.PropertyType
+import models.{PropertyType, RentalsRentARoom, UserAnswers}
 import pages.PageConstants.aboutPath
 import pages.QuestionPage
+import pages.adjustments._
+import pages.allowances.BusinessPremisesRenovationPage
+import pages.rentalsandrentaroom.adjustments.BusinessPremisesRenovationAllowanceBalancingChargePage
 import play.api.libs.json.JsPath
 
+import scala.util.{Failure, Success, Try}
+
 case class ClaimPropertyIncomeAllowancePage(propertyType: PropertyType) extends QuestionPage[Boolean] {
 
   override def path: JsPath = JsPath \ aboutPath(propertyType) \ toString
 
   override def toString: String = "claimPropertyIncomeAllowanceYesOrNo"
+
+  override def cleanup(value: Option[Boolean], userAnswers: UserAnswers): Try[UserAnswers] =
```

Comment:
No logic here like value is false?

Reply:
I am cleaning the old values because it means the user has decided to select the other option - it's either PIA or non-PIA